client/src/app/management/components/meeting-list/meeting-list.component.html

```diff
@@ -2,7 +2,7 @@
     [customMenu]="true"
     prevUrl="/committees"
     [nav]="false"
-    [hasMainButton]="true"
+    [hasMainButton]="operator.hasCommitteePermissions(currentCommittee?.id, CML.can_manage)"
```

Comment:
as I see the `?`: Please explicitly add `null|undefined` to the method in the operator and handle null cases explicitly. I know that it currently works, but handling these things explicitly gives clearance in the behavior.

Reply:
yes, this is right, but if you give `null` or `undefined`, it should return false.